tools/bin/check-duplicated-storages.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
-backup_storage_output=$(kubectl get backupstorage -n everest-system)
-backup_storage_names=($(echo "$backup_storage_output" | awk 'NR>1 {print $1}'))
+backup_storage_names=($(kubectl get backupstorage -n everest-system --no-headers -o=jsonpath='{.items[*].metadata.name}'))
+
+if ! command -v kubectl &> /dev/null; then
+  echo "Error: kubectl is not installed. Please install it first."
+  exit 1
+fi
```

Comment:
This needs to be moved a couple of lines up :joy: you're using kubectl in line 3

Reply:
😂  right, thanks for pointing that out